lib/logger.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+var Log;
+
+Log = require("log");
+
+module.exports = (logLevel) => {
+    var logger, logLevels;
+
+    logLevels = ["info", "debug", "warning"];
```

Comment:
Ah! Yeah, that makes a lot of sense. I should start doing this everywhere. Noted! And fixed.